api/src/test/java/org/itsallcode/openfasttrace/api/core/TestLinkedSpecificationItem.java

```diff
@@ -83,49 +84,58 @@ void testLinkWithStatus()
     @Test
     void testGetCoveredArtifactTypes()
     {
-        this.linkedItem.addCoveredArtifactType(UMAN);
-        this.linkedItem.addCoveredArtifactType(REQ);
+        when(this.coveredItemMock.getArtifactType()).thenReturn(UMAN);
+        this.linkedItem.addLinkToItemWithStatus(coveredLinkedItem, LinkStatus.COVERED_SHALLOW);
```

Comment:
This code makes no sense. Mock and real item mixed.

Reply:
? To my understanding this tests is meant to tests that addLinkToItemWithStatus adjusts coveredArtifactTypes cache of LinkedSpecificationItem. For that no fully fledged covered item is needed.